fhttp/http_client.go

```diff
@@ -1087,9 +1088,9 @@ func (d *DelayedErrorReader) Close() error {
 
 // Response reading:
 //
-//nolint:nestif,funlen,gocognit,gocyclo,maintidx,gosec // TODO: refactor - unwiedly/ugly atm.
+//nolint:nestif,funlen,gocognit,gocyclo,maintidx // TODO: refactor - unwiedly/ugly atm.
```

Comment:
and it's now more ugly... probably should refactor the types and/or remember the len->int64 conversions